sim/src/main/cc/midasexamples/TestHarness.h

```diff
@@ -65,6 +67,10 @@ class TestHarness : public simif_peek_poke_t, public simulation_t {
     run_test();
     return teardown();
   }
+
+protected:
+  /// Random number generator for tests, using a default seed.
```

Comment:
Just to confirm, default here means a fixed default, right?

Reply:
If so, go ahead and merge.

Reply:
Yes.